src/utils/db.js

```
@@ -0,0 +1,9 @@
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
```

Consider validating the DB connection at startup (e.g., `await client.authenticate()` or an exported `init` that calls `authenticate()`/`sync()` in a controlled way). This helps fail fast when DB credentials are wrong instead of getting runtime errors later.